Main.java

```diff
@@ -1,5 +1,7 @@
+import java.util.Scanner;
+
 /**
- *
+ *This program will convert the temperature in degrees Celsius into degrees Farenheit 
  * @author 

```

Comment:
Don't forget your name! How would someone ever attribute your work if you don't include it?!